mailbox/api/src/test/java/org/apache/james/mailbox/model/AttachmentTest.java

```diff
@@ -20,73 +20,18 @@
 
 package org.apache.james.mailbox.model;
 
-import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
-import java.io.InputStream;
-import java.nio.charset.Charset;
-import java.nio.charset.StandardCharsets;
-
-import org.apache.commons.io.IOUtils;
 import org.junit.jupiter.api.Test;
 
 class AttachmentTest {
-
-    private static Charset CHARSET = StandardCharsets.UTF_8;
-/*
-    @Test
-    void streamShouldBeConsumedOneTime() throws Exception {
-        String input = "mystream";
-        Attachment attachment = Attachment.builder()
-                .bytes(input.getBytes(CHARSET))
-                .type("content")
-                .build();
-
-        InputStream stream = attachment.getStream();
-        assertThat(stream).isNotNull();
-        assertThat(IOUtils.toString(stream, CHARSET)).isEqualTo(input);
-    }
-
-    @Test
-    void getByteShouldReturnByteArrayRepresentingTheAttachment() {
-        String input = "mystream";
-        Attachment attachment = Attachment.builder()
-            .bytes(input.getBytes(CHARSET))
-            .type("content")
-            .build();
-
-        byte[] bytes = attachment.getBytes();
-        assertThat(new String(bytes, CHARSET)).isEqualTo(input);
-    }
-
-    @Test
-    void streamShouldBeConsumedMoreThanOneTime() throws Exception {
```

Comment:
did you move this test elsewhere?

Reply:
You can load the attachment stream as needed.